doc/vscode.md

```diff
@@ -18,3 +18,43 @@ sudo apt install npm node-typescript
 
 _note_: as of today the vscode mode requires a full lambdapi install,
 it won't unfortunately run from a developer tree.
+
+# Overview
+
+This extension provides syntax highlighting, unicode characters easy insertion and proof navigation as in CoqIde or ProofGeneral.
+
+It is still under development, thus might be incomplete and buggy.
+
+# Features
+
+
+##  Proof navigation
+
+Goals are visualised in a panel on the right side of the editor.
+
+- You can navigate the proof with [keybindings](#keybindings).
+- 
+- "Cursor mode" can be toggled to navigate the proff with the cursor.
+
+##  Snippets
+
+Typing "@letter" will suggest a corresponding unicode greek letter (for instance "@b" will suggest β). Press Enter or Tab to confirm adding the Unicode character.
+
+# Keybindings
+
+- Navigate down : go forward in the proof
```

Comment:
Perhaps it is better to present things the other way around as follows:
key combination: action
```
- Ctrl-Down: go one step forward
- Ctrl-Up: go one step backward
- Ctrl-Enter: go to where is the cursor
- Ctrl-Alt-c: toggle cursor mode (follow the cursor or not)
```
By the way, is it Ctrl-Alt-c or Ctrl-Alt-C?